kids-chatbot/src/components/AdminLogin.tsx

```
@@ -0,0 +1,88 @@
+    }
+  }
+
+  return (
+    <div className="bg-white rounded-2xl shadow-lg p-8">
+      <form onSubmit={handleSubmit} className="space-y-6">
+        <div>
+          <label className="block text-sm font-medium text-gray-700 mb-2">
+            نام کاربری
+          </label>
+          <input
+            type="text"
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)